pkg/processors/collector/log/elasticsearch_collector.go

```diff
@@ -0,0 +1,243 @@
+package log
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	elasticsearch "github.com/elastic/go-elasticsearch/v7"
+	"github.com/go-logr/logr"
+
+	"github.com/kubediag/kubediag/pkg/processors"
+	"github.com/kubediag/kubediag/pkg/processors/utils"
+)
+
+const (
+	elasticsearchAddress  = "param.collector.log.elasticsearch.address"
```

Comment:
These keys should be public since others might develop processors depending on elasticsearch collector.